lively.ide/components/reconciliation.js

```diff
@@ -565,7 +565,7 @@ export function applyModuleChanges (reconciliation, scope, system, sourceEditor
 
     if (patchTextMorph) {
       const browser = sourceEditor.owner;
-      if (browser) browser.resetChangedContentIndicator();
+      if (browser.isBrowser) browser.resetChangedContentIndicator();
```

Comment:
Shouldn't this be `browser && browser.isBrowser`? If browser could have been falsy previously, this risks accessing `isBrowser` on undefined, no?